linker_scripts/linker_script_extra.ld

```diff
@@ -0,0 +1,7 @@
+ENTRY(entrypoint);
```

Comment:
I'd probably just say `extra.ld` for the name, though perhaps `assert.ld`, `entry.ld`, or `check.ld`?